Marlin/src/HAL/HAL_LPC1768/HAL.cpp

```diff
@@ -57,4 +58,23 @@ int16_t PARSED_PIN_INDEX(const char code, const int16_t dval) {
   return ind > -2 ? ind : dval;
 }
 
+// HAL idle task
+void HAL_idletask(void) {
```

Comment:
probably best to define this in main.cpp

Reply:
Wasn't sure which file made most sense, I think the Due has it in hal.cpp, but main may make more sense for the LPC1768.